### PR TITLE
Test Pococpp after excluding broken

### PR DESCRIPTION
**Pull request by shekharhimanshu**
Removed broken toolchains.
https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2791

### CODE REVIEW

**Comment by shekharhimanshu** [comment]
The diff doesn't look good. Will close and confirm again.

**Comment by shekharhimanshu** [comment]
I have removed broken toolchains. Running tests now.

File: jenkins.py [hunk 1]
```diff
@@ -104,6 +104,13 @@ def run():
           )
       )
 
+  if (
+      os.getenv('TRAVIS') and
+      (project_dir == 'examples/PocoCpp') and
+      toolchain.startswith('ios-nocodesign')
+  ):
+    verbose = False
```
Comment (ruslo) [inline jenkins.py]: You can add `VERBOSE=0` instead: https://github.com/ingenue/hunter/blob/7cb1740cca46b62739bd7df44a6fa8a5483dec4b/.travis.yml#L65-L66

**Review by ruslo — CHANGES_REQUESTED** [review]

File: jenkins.py (hunk 1 above)
Comment (shekharhimanshu) [reply jenkins.py]: To confirm, you want me to add `VERBOSE=0` in each test `env` inside `.travis.yml` file?

Comment (ruslo) [reply jenkins.py]: Not to every, only for iOS (only for trouble ones).

Comment (shekharhimanshu) [reply jenkins.py]: Added, https://github.com/ingenue/hunter/pull/163/commits/4f044ed8ecb272449d6d5d5229b90469e34c9bc8

Now testing ...

Comment (shekharhimanshu) [reply jenkins.py]: One of the OSX toolchains failed with
`The job exceeded the maximum time limit for jobs, and has been terminated.` error.

Comment (ruslo) [reply jenkins.py]: > The job exceeded the maximum time limit for jobs

Restarted

Comment (ruslo) [reply jenkins.py]: Please remove this since it's not needed. Then do rebase.

**Review by ruslo — CHANGES_REQUESTED** [review]

**Comment by shekharhimanshu** [comment]
Remove `verbose =false` and merged pkg.template branch.

**Comment by ruslo** [comment]
> merged pkg.template branch

No need to do that. If you want to create pull request for branch `pkg.pococpp` then start your branch from `ci/pkg.pococpp`:
* https://docs.hunter.sh/en/latest/creating-new/update.html#testing

I see conflicts now, please resolve.

**Comment by shekharhimanshu** [comment]
>No need to do that. If you want to create pull request for branch pkg.pococpp then start your branch from ci/pkg.pococpp

My bad. Will fix the conflicts.

**Review by ruslo — APPROVED** [review]

**Pull request merged**